### PR TITLE
Jb54269

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last commit belongs to this PR

File: src/tools/sfdk/sdkmanager.cpp [hunk 1]
```diff
@@ -1387,9 +1387,13 @@ bool SdkManager::stopReliably(VirtualMachine *virtualMachine)
 {
     QTC_ASSERT(virtualMachine, return false);
     bool ok;
+    if (isRunningReliably(virtualMachine)) {
```
Comment (vigejolla) [inline src/tools/sfdk/sdkmanager.cpp:1390]: As far as I understand, if the vm is just starting, isRunningReliably returns false. Should we care?

Comment (martyone) [reply src/tools/sfdk/sdkmanager.cpp:1390]: Actually isRunningReliably and the related isEngineRunning, isEmulatorRunning etc only care about the basic VM state. sfdk (the CLI) makes as much assumptions as possible in order to minimize startup time, so it never tries to really connect to the VM in the LBYL manner and instead it works with the basic VM state as long as possible.

Comment (vigejolla) [reply src/tools/sfdk/sdkmanager.cpp:1390]: I think I was confused by the excessive amounts of negations in the logic. I'm just going to assume that it's ok.

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**